test/hotspot/jtreg/TEST.groups

```diff
@@ -62,17 +62,33 @@ hotspot_handshake = \
 hotspot_serviceability = \
   serviceability
 
+# The resourcehogs tests might require a lot of resources  and
+# shouldn't be executed concurrently with ANY other tests
+# So the group shouldn't be part of any other group
 hotspot_resourcehogs = \
   resourcehogs
 
+# The other tests includes all tests that don't
+# test hotspot itself
+non_hotspot_tests = \
+  testlibrary_tests \
+  sources
+
+# Gtest unit-test with jtreg wrapper, VM flags are set by test
+hotspot_gtest = \
+ gtest
+
 hotspot_misc = \
```

Comment:
What is actually left in this group after all the listed excluded?

Reply:
Really, it contains only few tests now.  However it is going to contain anything that we might miss in the component and other specified groups.
Sometimes it would be makes sense to rename it to hotspot_empty and verify that it has only expected tests during source testing, I think.